little amendment of the example in test::read_body documentation

[code snippet](https://github.com/actix/actix-web/blob/9f196fe5a53fd1de6417bcdc68389fd24360cb20/src/test.rs#-L174-L191) doesn't compile.

## Current Behavior

```
cargo check --tests
...
error[E0369]: binary operation `==` cannot be applied to type `impl std::future::Future`
  --> src/main.rs:26:5
   |
26 |     assert_eq!(result, Bytes::from_static(b"welcome!"));
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |     |
   |     impl std::future::Future
   |     bytes::bytes::Bytes
```
## Possible Solution

```
diff --git a/src/test.rs b/src/test.rs
index 0eb02ff7..15e66a23 100644
--- a/src/test.rs
+++ b/src/test.rs
@@ -187,7 +187,7 @@ where
 ///         .to_request();
 ///
 ///     let resp = test::call_service(&mut app, req).await;
-///     let result = test::read_body(resp);
+///     let result = test::read_body(resp).await;
 ///     assert_eq!(result, Bytes::from_static(b"welcome!"));
 /// }
 /// ```
```

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->

* Rust Version `rustc 1.42.0 (b8cedc004 2020-03-09)`
* Actix Web Version:  2.0

--- a/src/test.rs
+++ b/src/test.rs
@@ -187,7 +187,7 @@ where
 ///         .to_request();
 ///
 ///     let resp = test::call_service(&mut app, req).await;
-///     let result = test::read_body(resp);
+///     let result = test::read_body(resp).await;
 ///     assert_eq!(result, Bytes::from_static(b"welcome!"));
 /// }
 /// ```

